### ISSUE TITLE
NullReferenceException in GameplayScreen when handling ipc on startup when ipc-state.txt has value "Ranking"

### ISSUE THREAD

**Issue by omkelderman** [ppy/osu#14532]
**Describe the bug:** 

lazer tournament client hangs on the black loading screen if you start it while `ipc-state.txt` in the osu stable folder has a content of "Ranking".

Fix is simple (just deleting the ipc-state.txt file or restarting osu stable/cuttingedge to have let it set to "Idle") but that doesnt help if you dont know that this is the issue. Lazer just hangs and doesnt give any indication of whats going on. Only by digging in runtime.log and looking in the code I was able to pin down the issue to a NullReferenceException on this line https://github.com/ppy/osu/blob/master/osu.Game.Tournament/Screens/Gameplay/GameplayScreen.cs#L167 due to `CurrentMatch.Value` having a value of `null`. Other than that the exception seems to be eaten silently? There is no indication in the UI that anything went wrong other than that nothing is happening at all.

**Screenshots or videos showing encountered issue:**
![image](https://user-images.githubusercontent.com/2544736/131190594-c7327457-60c9-4bc3-a48c-7b117182bb12.png)

**osu!lazer version:** 
2021.820.0-lazer

**Logs:**
[runtime.log](https://github.com/ppy/osu/files/7069490/runtime.log)

tldr set ipc-state.txt to a value of "Ranking" start lazer in tourney mode with a debugger attached and watch what happens

**Comment by peppy** [comment]
Did you attach the wrong log file? There's no trace of any error in it.

**Comment by omkelderman** [comment]
oh uhm, yes, sorry, I seemed to have uploaded the wrong file, I tested it a couple times with various ipc-state.txt values and I probably accidently uploaded one where it went ok. I went straight to bed after I opened this issue so I didnt see it till now.

Anyways, looks like you figured it out anyway. Thanks for the fix! <3
